### ISSUE TITLE
Request is forwarded with JSON payload and XML MIME Type

### ISSUE THREAD

**Issue by dmoranj** [telefonicaid/fiware-orion#703]
When a Context Provider is created with the following request:
```
POST /NGSI9/registerContext HTTP/1.1
fiware-service: smartGondor
fiware-servicepath: /gardens
host: 192.168.56.101:1026
accept: application/json
content-type: application/json
content-length: 250
Connection: keep-alive

{"contextRegistrations":[{"entities":[{"type":"Light","isPattern":"false","id":"TestClient:Light"}],"attributes":[{"name":"luminescence","type":"Lumens","isDomain":"false"}],"providingApplication":"http://192.168.56.1:4041/NGSI10"}],"duration":"P1M"}
```

and the followign `updateContext` is sent:
```
POST /NGSI10/updateContext HTTP/1.1
fiware-service: smartGondor
fiware-servicepath: /gardens
host: 192.168.56.101:1026
accept: application/json
content-type: application/json
content-length: 175
Connection: keep-alive

{"contextElements":[{"type":"Light","id":"TestClient:Light","isPattern":"false","attributes":[{"name":"luminescence","type":"integer","value":"18"}]}],"updateAction":"UPDATE"}
```

The request forwarded to the context provider arrives with the following headers:
```
    "user-agent": "orion/0.18.0-next libcurl/7.19.7",
    "host": "192.168.56.1:4041",
    "fiware-service": "smartgondor",
    "fiware-servicepath": "/gardens",
    "accept": "application/xml, application/json",
    "content-length": "319",
    "content-type": "application/xml"
```
and the following body:
```
{
  "contextElements" : [
    "contextElement" : {
      "type" : "Light",
      "isPattern" : "false",
      "id" : "TestClient:Light",
      "attributes" : [
        {
          "name" : "luminescence",
          "type" : "integer",
          "value" : "18"
        }
      ]
    }
  },
  "updateAction" : "UPDATE"
}
```

when they should arrive either in XML with `application/xml` MIME Type or in JSON with `application/json` MIME Type.

(The first two requests were sniffed directly from the Context Broker machine with tshark, while the forwarded request was read from the debugger in the client server).

**Comment by fgalan** [comment]
This is related with https://github.com/telefonicaid/fiware-orion/issues/722. Once #722 gets solved, probably the present issue gets solved also.

**Comment by kzangeli** [comment]
The bug of mixing mime type and payload was fixed in PR #721.
Being able to forward in either XML or JSON was fixed in PR #899.
